queue/private.php

```diff
@@ -111,7 +116,7 @@
 						 	</form>
```

Comment:
This form still has some bugs. For example, it doesn't clear when I switch between clients. Also, I can still type more than 255 characters into the field. The textarea has ng-maxlength="255" but it also needs maxlength="255". This issue doesn't match the scope of this ticket though. I just wanted to make note of it.

Reply:
Created https://github.com/UNL-CSE-AMBASSADORS/VITA/issues/152